frontend/packages/kubevirt-plugin/src/k8s/patches/vm/vm-cdrom-patches.ts

```diff
@@ -60,7 +60,7 @@ export const getCDsPatch = (
       const disk: CD = {
         name,
         bootOrder: existingCD ? bootOrder : newBootOrder,
-        cdrom: { bus: bus || 'virtio' },
+        cdrom: { bus: bus || 'scsi' },
```

Comment:
can you use `DiskBus.SCSI.getValue()` instead?

Reply:
Sorry. I didn't know we had an enum for that. That's much better, of course. =)